smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/endpointsV2/RuleSetParameterFinder.java

```diff
@@ -73,31 +73,6 @@ public Map<String, String> getClientContextParams() {
         return map;
     }
 
-    /**
-     * "The staticContextParams trait defines one or more context parameters that MUST
-     * be bound to the specified values. This trait MUST target an operation shape."
-     */
-    public Map<String, String> getStaticContextParams(Shape operationInput) {
```

Comment:
may need to mark the class with `@SmithyInternalApi` if removing a public method

Reply:
Verified that all other files which call `RuleSetParameterFinder` have `@SmithyInternalApi`

Added the mark in [`a0d25c5` (#1359)](https://github.com/smithy-lang/smithy-typescript/pull/1359/commits/a0d25c58441b9066700d74310e8f80605096c7ff)